benchexec/tablegenerator/columns.py

```diff
@@ -259,13 +280,24 @@ def _format_number(number, initial_value_sig_digits, number_of_significant_digit
 
     # Round to the given amount of significant digits
     intended_digits = min(initial_value_sig_digits, number_of_significant_digits)
-    if number != 0:
+    if number == 0:
+        formatted_value = '0'
+        if max_digits_after_decimal > 0 and initial_value_sig_digits > 0:
+            formatted_value += '.' + '0' * min(max_digits_after_decimal, initial_value_sig_digits)
+
+    elif isnan(number):
+        formatted_value = 'NaN'
+    elif isinf(number):
+        formatted_value = 'Inf'
```

Comment:
This looks like it converts all infinities to 'Inf', i.e., positive infinity, doesn't it?

Reply:
Yupp, thanks for the hint!